src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,72 @@
+package duke.ui;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.Scanner;
+
+/**
+ * Represents the user interface of the application.
+ */
+public class Ui {
+    private static final String SEP = "\t__________________________________________";
+    private final Scanner in;
+    private final PrintStream out;
```

Comment:
private final Constants should be in uppercase, with these classes names as nouns

```suggestion
    private final Scanner SCANNER_IN;
    private final PrintStream PRINTSTREAM_OUT;
```